modules/programs/kitty.nix

```diff
@@ -218,6 +210,20 @@ in {
       '';
     };
 
+    home.activation.kittythemechecker = mkIf (cfg.theme != null)
+      (hm.dag.entryAnywhere ''
+        theme_path=$(${pkgs.jq}/bin/jq '.[] | select(.name=="${cfg.theme}" | .file' ${pkgs.kitty-themes}/share/kitty-themes/themes/themes.json)
+        if [ -z "$theme_path" ]; then
+          echo 'kitty-themes does not contain a theme named ${cfg.theme}' &>2
+          exit 1
+        fi
+
+        if [ ! -f "${pkgs.kitty-themes}/share/kitty-themes/themes/$theme_path" ]; then
```

Comment:
This check might be redundant (I think this is testing that `kitty-theme`s is packaged correctly). Should I leave this in?